frontend/src/static/js/pages/styles/PlaylistPage.scss

```
@@ -486,6 +493,257 @@
+						position:relative;
+						display:block;
+						padding-right:$button-width;
+						border:1px solid var(--popup-hr-bg-color);
+						border-radius:2px;
+					}
+
+					input[type="text"]{
+						width:100%;
+						height:42px;
+						padding:1px 0 1px 16px;
+						font-family:Arial;
```

Stylelint (17.6.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)